tests/phpunit/tests/import/parser.php

```diff
@@ -16,7 +16,7 @@ function setUp() {
 			define( 'WP_LOAD_IMPORTERS', true );
 
 		if ( ! file_exists( DIR_TESTDATA . '/plugins/wordpress-importer/wordpress-importer.php' ) ) {
-			$this->fail( 'This test requires the WordPress Importer plugin to be installed in the test suite. See: https://make.wordpress.org/core/handbook/contribute/git/#unit-tests' );
+			$this->fail( 'This test requires the ClassicPress Importer plugin to be installed in the test suite. See: https://make.wordpress.org/core/handbook/contribute/git/#unit-tests' );
```

Comment:
ClassicPress Importer plugin has not yet been developed, maybe we should leave this as is for now? If we change it, shouldn't the link on L18 then not also change?